newsfragments/270.feature.md

```diff
@@ -0,0 +1,29 @@
+Support for the boolean operations `and` and `or`.
+
+Examples:
+
+```
+contract Foo:
+    pub def bar(x: bool, y: bool) -> bool:
+        return x and y
+```
+
+```
+contract Foo:
+    pub def bar(x: bool, y: bool) -> bool:
+        return x or y
+```
+
+Support for `self.address`.
+
+This expression returns the address of the current contract.
+
+Example:
+
+```
+contract Foo:
+    pub def bar() -> address:
+        return self.address
```

Comment:
We should put `address` into the keywords section of the spec

https://github.com/ethereum/fe/blob/5142f7e9c68aa6b5b3df6e2a85c3ab212d8badbc/spec/index.md#21-keywords